### PR TITLE
chapter_13

### PR DESCRIPTION
**Pull request by azimfaiz**

### CODE REVIEW

**Comment by azimfaiz** [comment]
ready

File: app/assets/stylesheets/microposts.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Microposts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (saobv-2223) [inline app/assets/stylesheets/microposts.scss]: remove file that never used

File: app/controllers/static_pages_controller.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
-
 class StaticPagesController < ApplicationController
-  def home; end
+  def home 
+    return if logged_in?
+    
+    @micropost = current_user.microposts.build
+    @feed_items = current_user.feed.paginate(page: params[:page]).per `number`
```
Comment (saobv-2223) [inline app/controllers/static_pages_controller.rb]:
```suggestion
    @feed_items = current_user.feed.paginate(page: params[:page]).per 123
```
move 123 to settings.yml file and call it here

File: app/views/shared/_micropost_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<%= form_for @micropost do |f| %>
+  <%= render "shared/error_messages", object: f.object %>
+  <div class="field">
+    <%= f.text_area :content, placeholder: "Compose new micropost..." %>
+  </div>
+  <%= f.submit "Post", class: "btn btn-primary" %>
+  <span class="image">
+    <%= f.file_field :image, accept: "image/jpeg,image/gif,image/png" %>
```
Comment (saobv-2223) [inline app/views/shared/_micropost_form.html.erb]:
```suggestion
    <%= f.file_field :image, accept: "image/jpeg, image/gif, image/png" %>
```

**Comment by azimfaiz** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
merge

**Pull request merged**